DIRECTOR_HANDBOOK.md

```diff
@@ -0,0 +1,92 @@
+
+Generally, the Board may take action at meetings at which the requisite number of Directors is participating (in person or via communications equipment), and the requisite number Directors vote in favor, in each case as provided in Article IV of the By-Laws. Actions may also be taken electronically in accordance with the rules of the same Article. In some cases, a Super Majority (two-thirds) of the then serving Directors is required to approve an action.
+
+Additional information about Board governance and Directors is contained in Article IV of the By-Laws, which all Directors should review carefully.
+
+## Core Responsibilities
+
+The Board’s core responsibilities consist of the following:
+
+* Advising and counseling management regarding significant issues facing the Foundation;
+* Evaluating and approving the Foundation’s strategic direction and initiatives, and monitoring its implementation and results;
+* Monitoring the Foundation’s activities and financial condition;
+* Overseeing the Foundation’s integrity and ethics, compliance with laws and financial integrity and reporting;
+* Succession planning and management development;
+* Assessing the performance of senior management and setting compensation, if applicable; and
+* Understanding and assessing risks to the Foundation and monitoring the management of those risks.
+
+To fulfill its core responsibilities, Directors are expected to attend all meetings (except when special circumstances or pre-established material commitments dictate otherwise) and review materials in advance of the meetings.
+
+## Director Duties and Responsibilities
+
+Directors have a fiduciary duty to the Foundation to act in the best interests of the Foundation and its Members as a whole, including but not limited to acting with _due care_ and owing a _duty of loyalty_ to the Foundation.
+
+Your _duty of care_, in general, means that you should be reasonably informed to participate in decisions of the Board and to do so prudently and in good faith. Important aspects of this duty include:
+
+* __Attendance at Meetings__: You should attend all meetings of the Board of Directors (except when special circumstances or pre-established material commitments dictate otherwise). In order to do its job, the Board must act as a group, so your attendance at meetings is critical to ensure that all voices are heard.
+* __Exercising Independent Judgment__: Directors serve as individuals, and the effectiveness of any vote or action they take as a member of the Board should not depend on whether it was authorized or directed by any other person or entity or was contrary to the instructions or interests of any other person or entity. Rather, you should weigh your votes on official action carefully, based upon your independent business judgment, and you should abstain or dissent when you believe any action for which approval is sought is not in the best interests of the Foundation or its mission.
```

Comment:
This should be clarified further along with a simple recognition that Directors do, in fact, where multiple hats, representing both the interests of the Foundation membership *and* either the interests of the constituency they represent (corporate members, individual members, etc). The important distinction is that while individual directors represent those interests, decisions are based on their individual business judgement based on their personal view of what is in the Foundation's best interests -- it just really needs to understood that such judgments are often formed in complex ways.